builder/ecs/run_config.go

```diff
@@ -51,7 +51,7 @@ type RunConfig struct {
 	// The default value is false.
 	DisableStopInstance bool `mapstructure:"disable_stop_instance" required:"false"`
 	// Ram Role to apply when launching the instance.
-	RamRoleName string `mapstructure:"ram_role_name" required:"false"`
+	RamRoleName string `mapstructure:"ecs_ram_role_name" required:"false"`
```

Comment:
This will break existing configs so I would advise not renaming it, but instead deprecating this one through a mention in the docs, and adding an extra `ecs_ram_role_name` to eventually replace it later.

Reply:
Hi @lbajolet-hashicorp,
Before [PR](https://github.com/hashicorp/packer-plugin-alicloud/pull/57) merged, ram_role_name only work for Alicloud API [CreateInstance](https://www.alibabacloud.com/help/en/elastic-compute-service/latest/createinstance) with parameter RamRoleName, it works right since April,2021. 
But since [PR](https://github.com/hashicorp/packer-plugin-alicloud/pull/57) merged, parameter ram_role_name works both for authentication with [NewClientWithEcsRamRole](https://github.com/aliyun/alibaba-cloud-sdk-go/blob/master/services/ecs/client.go) and API CreateInstance. Because of this change, user feedback that their existing templates with ram_role_name not work since packer plugin updated last week.

Reply:
Hi @alexyueer,

Sorry to say that, but I'm a bit confused still, the PR you refer to introduces this attribute to the configuration, so it should not have been part of your client's configs before as it did not exist.
This PR has been merged in August last year, and was introduced in v1.0.3 of the plugin, there are three more releases since then, was there another change elsewhere that broke this? Or is it an API change on the Alicloud side of things that introduces this regression?